src/transformers/data/data_collator.py

```diff
@@ -166,8 +264,23 @@ class DataCollatorForTokenClassification:
     max_length: Optional[int] = None
     pad_to_multiple_of: Optional[int] = None
     label_pad_token_id: int = -100
+    return_tensors: str = "pt"
+
+    def __call__(self, features, return_tensors=None):
+        if return_tensors is None:
+            return_tensors = self.return_tensors
+        if return_tensors == "pt":
+            return self.torch_call(features)
+        elif return_tensors == "tf":
+            return self.tf_call(features)
+        elif return_tensors == "np":
+            return self.numpy_call(features)
+        else:
+            raise ValueError(f"Framework '{return_tensors}' not recognized!")
```

Comment:
This could be refactored in a MixIn to avoid being in almost all the classes here.

Reply:
Done!